src/decode.rs

```diff
@@ -4267,6 +4351,12 @@ pub(crate) unsafe fn rav1d_decode_frame_init(
             return Err(ENOMEM);
         }
         f.n_ts = n_ts;
+
+        for index in 0..n_ts as usize {
+            let ts = &mut *f.ts.add(index);
+            // TODO: Safe initialization.
+            addr_of_mut!(ts.context).write(Mutex::new(Default::default()));
```

Comment:
It's not sound to have the intermediate `&mut Rav1dTileState` here if it's uninitialized, but since this gets fixed right away in #1032, it's fine here.